### PR TITLE
Feature/519 text plain

### LINKED ISSUE #519: Add plain/text format to Clob end-point that skips the JSON encapsulation and returns the raw text.
This feature would be used by text-timeseries and async retrieval if it was present.

### PR DESCRIPTION
**Pull request by rma-rripken**
closes #519

### CODE REVIEW

File: cwms-data-api/src/main/java/cwms/cda/api/ClobController.java [hunk 1]
```diff
@@ -156,19 +155,27 @@ public void getAll(@NotNull Context ctx) {
...
 
 
     @OpenApi(
             queryParams = {
                     @OpenApiParam(name = OFFICE, description = "Specifies the owning office."),
+                    @OpenApiParam(name = CLOB_ID, description = "If this _query_ parameter is provided the id _path_ parameter "
+                            + "is ignored and the value of the query parameter is used.   "
+                            + "Note: this query parameter is necessary for id's that contain '/' or other special "
+                            + "characters.  Because of abuse even properly escaped '/' in url paths are blocked.  "
+                            + "When using this query parameter a valid path parameter must still be provided for the request"
```
Comment (MikeNeilson) [inline cwms-data-api/src/main/java/cwms/cda/api/ClobController.java:169]: It's actually a slight failure in Javalin in how it even processes the paths... there's a section that just, doesn't.

But I like this wording.

Comment (MikeNeilson) [reply cwms-data-api/src/main/java/cwms/cda/api/ClobController.java:169]: Well, and could be both.

File: cwms-data-api/src/test/java/cwms/cda/api/ClobControllerTestIT.java [hunk 2]
```diff
@@ -69,26 +64,41 @@ void test_create_getOne() throws JsonProcessingException, UnsupportedEncodingExc
             .assertThat()
             .statusCode(is(HttpServletResponse.SC_CREATED));
 
-        /* There is an issue with how javalin handles / in the path that are actually part
-        of the object name (NOTE: good candidate for actually having a GUID or other "code"
-        as part of the path and the actual name as a query parameter.
-        )
+//        /* There is an issue with how javalin handles / in the path that are actually part
```
Comment (MikeNeilson) [inline cwms-data-api/src/test/java/cwms/cda/api/ClobControllerTestIT.java:67]: Odd additional line comments vs block comments.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Looks reasonable to me. Just some odd, probably automatic, formatting to fix before it gets merged in.

File: cwms-data-api/src/main/java/cwms/cda/api/ClobController.java (hunk 1 above)
Comment (rma-rripken) [reply cwms-data-api/src/main/java/cwms/cda/api/ClobController.java:169]: what could be both?

File: cwms-data-api/src/test/java/cwms/cda/api/ClobControllerTestIT.java (hunk 2 above)
Comment (rma-rripken) [reply cwms-data-api/src/test/java/cwms/cda/api/ClobControllerTestIT.java:67]: yeah, that is odd.

File: cwms-data-api/src/main/java/cwms/cda/api/ClobController.java (hunk 1 above)
Comment (MikeNeilson) [reply cwms-data-api/src/main/java/cwms/cda/api/ClobController.java:169]: Bad processing by Javalin (I checked)
But also the security issue. I know that encoding in the path can cause issues, but even after I turned off that check it still doesn't work even though the manual says it should.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**